core/src/test/java/io/grpc/internal/AbstractClientStream2Test.java

```diff
@@ -109,14 +110,14 @@ public void cancel_failsOnNull() {
 
   @Test
   public void cancel_notifiesOnlyOnce() {
-    final BaseTransportState state = new BaseTransportState();
+    final BaseTransportState state = new BaseTransportState(statsTraceContext);
```

Comment:
sed'ed and shorted for variables named statsTraceContext, censusContext and censusContextFactory.